src/main/java/org/spongepowered/api/keyboard/PlayerKeyBinding.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.keyboard;
+
+import org.spongepowered.api.entity.living.player.Player;
+
+import java.util.Optional;
+
+/**
+ * Represents a {@link KeyBinding} held by a specific {@link Player}.
+ */
+public interface PlayerKeyBinding {
```

Comment:
@Zidane 
The main purpose of this class is to give direct access of the state of a key binding for a specific player. The `setEnabled` and `getEnabled` methods can also only be accessed through this class.
Maybe are the difference in names and purpose between PlayerKeyBinding and KeyBinding a bit confusing, but that was the only good name I could find at that time.